### PR TITLE
PIE-2527] Add information regarding flaky summary mailer to assignments page

### PR DESCRIPTION
**Pull request by KatieWright26**
Screenshot 2024-07-01 at 12 32 37 (2)

[Relevant linear](https://linear.app/buildkite/issue/PIE-2527/document-notification-feature)

### CODE REVIEW

File: pages/test_analytics/flaky_test_assignment.md [hunk 1]
```diff
@@ -34,3 +34,8 @@ When an assigned test has not flaked in more than 7 days, it is moved to the **O
 
 <%= image "outdated-assignments.png", width: 1960/2, height: 630/2, alt: "Flaky test page showing team assignments" %>
 
+## Weekly flaky test mailer
```
Comment (jasmine-q) [inline pages/test_analytics/flaky_test_assignment.md:37]: The word mailer doesn't feel right to me here, maybe "Weekly flaky test summary report"? Or "Weekly flaky test summary"

Comment (KatieWright26) [reply pages/test_analytics/flaky_test_assignment.md]: Cool, will change to Weekly flaky test summary

File: pages/test_analytics/flaky_test_assignment.md [hunk 2]
```diff
@@ -34,3 +34,8 @@ When an assigned test has not flaked in more than 7 days, it is moved to the **O
 
 <%= image "outdated-assignments.png", width: 1960/2, height: 630/2, alt: "Flaky test page showing team assignments" %>
 
+## Weekly flaky test summary
+
+You're able to schedule a weekly summary of the flakiest tests assigned to your teams. Visit the **Suite settings** page to create new notifications, or manage existing ones. If you would like to set up auto assignment, check out our [Test ownership](/docs/test-analytics/test-ownership) feature.
```
Comment (meghan-kradolfer) [inline pages/test_analytics/flaky_test_assignment.md:39]: Should there be anything about this being available only for pro? Or is this implied from the connection to flaky assignments?

Comment (KatieWright26) [reply pages/test_analytics/flaky_test_assignment.md:39]: This is part of the assignments page which says at the top that it's a paid feature, so I don't think we need to reiterate here

**Review by meghan-kradolfer — APPROVED** [review]

**Pull request merged**